### ISSUE TITLE
POST-ing the same config twice will fail on subsequent attempts.

### ISSUE THREAD

**Issue by LoganM-F5** [F5Networks/f5-appsvcs-extension#496]
### Environment
 * Application Services Version: 3.29.0
 * BIG-IP Version: 15.1.2.1

### Summary
POST-ing the same declaration twice will fail if the declaration uses a non-default route domain and has a period "." in its name.
POST-ing an equivalent declaration with an underscore "_" instead will work without issue.
Working:
```json
"dot_test": {
    "class": "Tenant",
    "defaultRouteDomain": 2,
```
Not Working
```json
"dot.test": {
    "class": "Tenant",
    "defaultRouteDomain": 2,
```
### Steps To Reproduce
1. Submit the following declaration twice:
```json
{
    "action": "deploy",
    "class": "AS3",
    "syncToGroup": "manual-config-sync",

    "declaration": {
        "class": "ADC",
            "controls": {
            "class": "Controls",
            "trace": true,
            "logLevel": "debug"
        },
        "dot.test": {
            "class": "Tenant",
            "defaultRouteDomain": 2,
            "dot_test_http": {
                "class": "Application",
                "dot_test_service_address": {
                    "class": "Service_Address",
                    "routeAdvertisement": "enable",
                    "virtualAddress": "10.204.64.249"
                },
                "dot_test_http": {
                    "class": "Service_HTTP",
                    "snat": "self",
                    "virtualAddresses": [
                        {
                            "use": "dot_test_service_address"
                        }
                    ],
                    "virtualPort": 443
                },
                "template": "generic"
            }
        },
        "schemaVersion": "3.29.0",
        "updateMode": "selective"
    },
    "persist": true
}
```

2. Observe the following error response for the second POST:
```json
{
    "code": 422,
    "message": "declaration failed",
    "response": "The Service Address virtualAddress property cannot be modified. Please delete /dot.test/dot_test_service_address and recreate it.",
    "host": "localhost",
    "tenant": "dot.test",
    "runTime": 2043
}
```

### Expected Behavior
The config should be identified as being the same and return an appropriate response.
```json
{
    "code": 200,
    "message": "no change",
    "host": "localhost",
    "tenant": "dot_test",
    "runTime": 1241
}
```
### Actual Behavior
The config does not match what was previously declared and returns an error message.
`"response": "The Service Address virtualAddress property cannot be modified. Please delete /dot.test/dot_test_service_address and recreate it."`

**Comment by dstokesf5** [comment]
Thank you for reporting this bug. I have added it to our internal product backlog as AUTOTOOL-2708.

**Comment by dstokesf5** [comment]
This has been resolved in the 3.32.0 release.
